frontend/src/MakerApp.tsx

```diff
@@ -66,16 +72,10 @@ export default function App() {
         },
     });
 
-    const runningStates = ["Accepted", "Contract Setup", "Pending Open"];
-    const running = cfds.filter((value) => runningStates.includes(value.state));
-    const openStates = ["Requested"];
-    const open = cfds.filter((value) => openStates.includes(value.state));
-    const closedStates = ["Rejected", "Closed"];
-    const closed = cfds.filter((value) => closedStates.includes(value.state));
-    // TODO: remove this. It just helps to detect immediately if we missed a state.
-    const unsorted = cfds.filter((value) =>
-        !runningStates.includes(value.state) && !closedStates.includes(value.state) && !openStates.includes(value.state)
-    );
+    const acceptOrReject = cfds.filter((value) => value.state.getGroup() === StateGroupKey.ACCEPT_OR_REJECT);
+    const opening = cfds.filter((value) => value.state.getGroup() === StateGroupKey.OPENING);
```

Comment:
I'm not sold on having an extra state/tab for `opening`. Imho the user does not care about the difference between `opening` and `open`.

Reply:
Yeah, I am not so sure about it neither - I think there are good arguments for and against it. 
I think we will have to see what actual users say :)